vendor/github.com/openshift/installer/data/data/bootstrap/systemd/units/fluentbit.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
```

Comment:
all these files are frustratingly similar to the content in pkg/genevalogging, but I don't think there's much to be done about it.  A comment in pkg/genevalogging drawing attention would be good, so that if people change one they think about changing the other?

Reply:
Added comment in genevalogging.go